arches_lingo/src/arches_lingo/components/header/DarkModeToggle.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import ToggleButton from "primevue/togglebutton";
+
+import { DEFAULT_THEME } from "@/arches/themes/default.ts";
+
+const { $gettext } = useGettext();
+
+const darkModeClass = DEFAULT_THEME.theme.options.darkModeSelector.substring(1);
```

Comment:
There's a little asymmetry worth deciding on.

In archesproject/arches#11667, I hard coded the `arches-dark` class name, but here I'm reading it from the default theme. The idea being that if these two ever drift (e.g. `my-arches-consultancy-dark`), it's N fewer things to update, just override base-root.htm. Thoughts?

Reply:
IMO we should strongly discourage ever overwriting base-root.htm. I"m not sure this pattern jives with if an application defines its own theme. However happy to punt those considerations until this toggle lives outside of Lingo